components/com_users/tmpl/methods/list.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_users
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Prevent direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Uri\Uri;
+use Joomla\Component\Users\Site\Model\MethodsModel;
+use Joomla\Component\Users\Site\View\Methods\HtmlView;
+
+// phpcs:ignoreFile
+/** @var HtmlView $this */
+
+/** @var MethodsModel $model */
+$model = $this->getModel();
+
+$this->document->getWebAssetManager()->useScript('com_users.two-factor-list');
+
+HTMLHelper::_('bootstrap.tooltip', '.hasTooltip');
+
+?>
+<div id="com-users-methods-list-container" class="p-1">
```

Comment:
It was removed in the admin, but skipped in the frontend.

```suggestion
<div id="com-users-methods-list-container">
```